### PR TITLE
Fix Bug 304 - Editor scrolls to bottom when pressing enter in the mid…

### PR DESCRIPTION
**Pull request by andpor**
**Summary**

Fixes issue #304 . Minor code change. No impact on any other code.

**Test Plan**

All tests pass. N

### CODE REVIEW

**Comment by bradbumbalough** [comment]
We are facing this same issue, however, I also needed to factor the `pOffset` value in first if handler as well (if `scrollParent === window`). It would be added to line https://github.com/facebook/draft-js/pull/1223/files#diff-26f7fbfbc0fd3c1e08d0d6b1b9bf21b8R105. 

@andpor Did you encounter that as well?

**Comment by andpor** [comment]
@bradbumbalough no I did not have this issue. What is the use case where this is failing so I can check as well on my side.

I can't believe this PR is still not merged....

**Comment by andpor** [comment]
Can someone form Facebook explain why the PR has not yet been merged. This is a serious showstopper issue. PR has been opened since May 27 - this is 4 months already.

**Comment by jackyho112** [comment]
As you mentioned [here](https://github.com/facebook/draft-js/issues/304#issuecomment-338156791), the issue seems to have been fixed. Feel free to close this PR. Thanks for the contribution!

**Comment by henrychung-bauerxcel** [comment]
@andpor @jackyho112 The issue has not been fixed with the overflow: auto, unfortunately. The changes in this PR does. Is anyone still looking in to this? Can this be merged? It's a one-liner and it does fix the issues described in https://github.com/facebook/draft-js/issues/304. Anyhow, it would make a whole lot of people happy if this could be resolved :)

**Comment by terenceodonoghue** [comment]
Still an issue as at 0.10.5 - any plans to merge this PR?

**Comment by niveditc** [comment]
Hey folks! Apologies for the delay in merging this PR.

@andpor - can you rebase on master & fix the conflicts. Then I'll merge it in :)

**Comment by niveditc** [comment]
@andpor - just wanted to ping this! I'd be happy to merge it in once the conflicts have been fixed.

**Comment by andpor** [comment]
After 2 years ???????

**Comment by andpor** [comment]
I think I resolved conflicts - please double check and merge.

**Comment by andpor** [comment]
@niveditc - please let me know what needs to be done to have this merged...

**Comment by Mrwaite** [comment]
> @niveditc - please let me know what needs to be done to have this merged...
After 2 years...... hope to  merge this PR as soon as possibile

**Comment by mrkev** [comment]
@andpor thanks for rebasing, importing to check it out.

File: src/component/contents/DraftEditorBlock.react.js [hunk 1]
```diff
@@ -121,8 +121,9 @@ class DraftEditorBlock extends React.Component<Props> {
         blockNode instanceof HTMLElement,
         'blockNode is not an HTMLElement',
       );
-      const blockBottom = blockNode.offsetHeight + blockNode.offsetTop;
-      const scrollBottom = scrollParent.offsetHeight + scrollPosition.y;
+      var blockBottom = blockNode.offsetHeight + blockNode.offsetTop;
+      var pOffset = scrollParent.offsetTop + scrollParent.offsetHeight;
+      var scrollBottom =  pOffset + scrollPosition.y;
```
Comment (claudiopro) [inline src/component/contents/DraftEditorBlock.react.js]: Hi @andpor, can you change these variable declarations to use `const` instead? Thanks!

Comment (andpor) [reply src/component/contents/DraftEditorBlock.react.js]: @claudiopro I just tried. Can you check how it looks?

**Comment by andpor** [comment]
What else needs to be done for this PR to get merged?

**Comment by andpor** [comment]
What is going on with this PR ?

**Comment by lintuming** [comment]
It's seems some lint error caused the Travis CI build failed,can you fix it.
image

**Comment by mrkev** [comment]
Fixed in https://github.com/facebook/draft-js/pull/2197

**Pull request closed without merging**